classes/author.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'item'
+
+class Author < Item
+  attr_accessor :id, :first_name, :last_name
+  attr_reader :items
+
+  def initialize(params)
+    super(params)
+
+    @id = params['id']
+    @first_name = params['first_name']
+    @last_name = params['last_name']
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end
```

Comment:
According to the project below requirements : 
**1- add_item method in the Author class
    - should take an instance of the Item class as an input
    - should add the input item to the collection of items
    - should add self as a property of the item object (by using the correct setter from the item object)**
It would be better to define the method as below, because we need to add the input item to the collection of items and it should add self as a property of the item object 😊

def add_item(item)
    return if @items.include?(item)
    @items << item
    item.add_author(self)
  end